src/audio.h

```diff
@@ -74,7 +74,7 @@ void music_jingle_voicelimit_set(u8 voiceLimit);
 void music_fade(s32 time);
 void music_volume_reset(void);
 void sound_update_queue(u8 updateRate);
-void sound_play_delayed(u16 soundId, SoundHandle *soundMask, f32 delayTime);
+void sound_play_delayed(u16 soundId, SoundHandle *handlePtr, f32 delayTime);
```

Comment:
nit-picky, BUT: why rename this to `handlePtr` instead of `soundHandle` like you did for the other rename below?

Reply:
nit-picky is ok. This was an automated change from the formatter to match what the function itself uses.